src/BlazorUI/Demo/Client/Core/Pages/Components/Spinner/BitSpinnerDemo.razor

```diff
@@ -32,7 +32,7 @@
         </ExamplePreview>
     </ComponentExampleBox>
 
-    <ComponentExampleBox Title="Label positioning" RazorCode="@example2RazorCode" Id="example2">
+    <ComponentExampleBox Title="Label Position" RazorCode="@example2RazorCode" Id="example2">
```

Comment:
from the issue's description:
```
This title should be changed to `Position`.
```

so please change the title from `Label positioning` to `Position` (the word `Label` should be removed).